### PR TITLE
Add featured-content index

### PR DESCRIPTION
**Pull request by pfrazee**
This adds the new featured-content index which is controlled by admin routes. This also puts the popular index behind and admins-only view.

### CODE REVIEW

**Comment by pfrazee** [comment]
screen shot 2017-06-07 at 7 35 47 pm
screen shot 2017-06-07 at 7 36 04 pm
screen shot 2017-06-07 at 7 36 15 pm

File: assets/css/pages/archive.less [hunk 1]
```diff
@@ -8,6 +8,21 @@ main.archive {
     }
   }
 
+  h1 .featured {
+    // bullshit CSS because tara needs to take a pass at this
```
Comment (taravancil) [inline assets/css/pages/archive.less]: lol

Comment (taravancil) [inline assets/css/pages/archive.less]: I plan to spend some time in the near future doing a bullshit CSS pass, so worry not

File: assets/css/pages/archive.less [hunk 2]
```diff
@@ -8,6 +8,21 @@ main.archive {
...
+    font-size: 0.8rem;
+    font-weight: normal;
+    border: 1px solid gray;
+    border-radius: 2px;
+    display: inline-block;
+    line-height: 1.2;
+    padding: 1px 4px 0;
+    background: #fafafa;
+    color: #666;
```
Comment (taravancil) [inline assets/css/pages/archive.less]: 😈

File: assets/html/archive.html [hunk 3]
```diff
@@ -92,19 +95,33 @@
           </button>
         </form>
       <% } %>
+
+      <% if (sessionUser && sessionUser.scopes.includes('admin:dats')) { %>
+        <div class="card admin-controls">
+          <h1>admin</h1>
```
Comment (taravancil) [inline assets/html/archive.html]: nitpick: only one `<h1>` per page

**Pull request merged**